sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -1222,6 +1225,57 @@ export interface SearchIndexerWarning {
     readonly name?: string;
 }
 
+// @public
+export class SearchIndexingBufferedSender<T> {
+    constructor(client: SearchClient<T>, options?: SearchIndexingBufferedSenderOptions);
+    // (undocumented)
```

Comment:
Please add ref docs to all the new types

Reply:
Done